apps/vs-code-intellisense/README.md

```diff
@@ -1,6 +1,6 @@
 # Twilio Paste Intellisense
 
-Twilio Paste Intellisense extends the development experience of Twilio Paste by giving Visual Studio Code users additional functionality such as hover.
+Twilio Paste Intellisense extends the development experience of Twilio Paste by giving Visual Studio Code users additional functionality.
```

Comment:
Note: I wanted the README to match the docs page on the website, so I updated this file to match